addon/templates/components/frost-password.hbs

```diff
@@ -3,6 +3,7 @@
   class='frost-password-input'
   disabled=disabled
   form=form
+  hook=(concat hook '-text')
```

Comment:
Actually, is that even necessary or do you just need to pass `hook=hook` and let `frost-text` append the `-input`?